doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+                    <ul class="nav navbar-nav">
+                        <!-- Add link to Homepage to training when created -->
+                        <li class="active"><a href="#">Home</a></li>
+                        <li class="dropdown">
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Fundamentals material <span class="caret"></span></a>
+                            <ul class="dropdown-menu">
+                                <li class="dropdown-header">Fundamentals Tutorials</li>
+                                <li><a href="#S1">Why use version control</a></li>
+                                <li><a href="#S2">svn</a></li>
+                                <li><a href="#S3">scenarios</a></li>
+                                <li><a href="#S4">configuration</a></li>
+                                <li role="separator" class="divider"></li>
+                                <li class="dropdown-header">Fundamentals Practical's</li>
+                                <li><a href="#S5">own repository</a></li>
+                                <li><a href="#S6">tutorial repositories</a></li>
+                            </ul>
+                        <li class="dropdown">
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Other material <span class="caret"></span></a>
+                            <ul class="dropdown-menu">
+                                <li class="dropdown-header">Quizzes</li>
+                                <li><a href="fundamentals_quiz">Fundamentals Quiz</a></li>
+                                <li><a href="essentials_quiz">Essentials Quiz</a></li>
+                                <li><a href="specifics_quiz">Specifics Quiz</a></li>
+                                <li role="separator" class="divider"></li>
+                                <li class="dropdown-header">Tutorials and practical's</li>
+                                <li><a href="fcm_essentials">Essentials</a></li>
+                                <li><a href="fcm_specifics">Specifics</a></li>
+                                <li role="separator" class="divider"></li>
+                                <li class="dropdown-header">Help/info</li>
+                                <li><a href="fcm_glossary">Glossary List</a></li>
+                                <li><a href="fcm_links">Links</a></li>
+                                <li><a href="fcm_mqrg">My Quick Reference Guide</a></li>
+                            </ul>
+                        </li>
+                    </ul>
+                </div>
+                <!--/.nav-collapse -->
+            </div>
+        </nav>
+        <div class="container theme-showcase" role="main">
+        <div class="jumbotron">
+            <h1>Fundamentals of version control, svn and fcm</h1>
```

Comment:
Needs to be "Subversion" not svn at this point as acronym hadn't yet been introduced.